plugin-support/woocommerce/_scss/woo.scss

```
@@ -39,3 +48,54 @@
     text-align: right;
     width: 5rem;
 }
+
+/*-------------------------------------------*/
```

Stylelint (17.10.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)